elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -0,0 +1,194 @@
+
+:host {
+  --pfe-navigation__overlay--BackgroundColor: var(--pfe-theme--overlay--BackgroundColor, rgba(37, 37, 37, 0.5));
+
+  // Default spacing values
+  --pfe-navigation--Padding--vertical: 20px; // calc(#{pfe-var(container-spacer)} * 4);
+  --pfe-navigation--Padding--horizontal: #{pfe-var(container-spacer)};
+
+  --pfe-navigation--Padding: 0 #{pfe-local(Padding--horizontal)};
+
+  // Darkest is the default background color
+  --pfe-navigation--BackgroundColor: #{pfe-color(surface--darkest)};
+  --pfe-navigation--Color: #{pfe-color(surface--darkest--text)};
+  --pfe-navigation--Border: #{pfe-var(surface--border-width)} #{pfe-var(surface--border-style)} transparent;
+  --pfe-navigation--Width: auto;
+  --pfe-navigation--MinHeight: 72px;
+
+  // Loop through the other breakpoints
+  @each $size in (md, lg, xl) {
+    @media screen and (min-width: pfe-breakpoint(#{$size})) {
+      --pfe-navigation--Width: calc(#{pfe-breakpoint(#{$size})} - calc(#{pfe-local(Padding--horizontal)} * 2));
+    }
+  }
+
+  // Reduce padding for print
+  @media print {
+    --pfe-navigation--Padding: calc(#{pfe-local(Padding--vertical)} / 2) #{pfe-local(Padding--horizontal)};
+  }
+
+  display: block;
+}
+
+:host(.sticky) {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  z-index: 99;
+}
+
+::slotted([slot="logo"]) {
+  margin: 0 !important;
+  padding-top: #{pfe-local(Padding--vertical)} !important;
+  padding-right: #{pfe-var(container-padding)} !important;
+  padding-bottom: #{pfe-local(Padding--vertical)} !important;
+  max-height: #{pfe-local(MinHeight)};
+}
+
+::slotted([slot="skip"]) {
+  @include pfe-sr-only;
+}
+
+::slotted([slot="main"]) {
+  display: none;
+  @media (min-width: #{pfe-breakpoint(lg)}) {
+    display: contents;
+  }
+}
+
+::slotted([slot="mobile-login"]),
+::slotted([slot="mobile-language"]) {
+  color: var(--pfe-broadcasted--color--ui-link, #{pfe-color(ui-link)});
+  text-decoration: var(--pfe-broadcasted--link--text-decoration, #{pfe-var(link--text-decoration)});
+}
+
+@each $icon in (user, globe) {
+  ::slotted([pfe-icon="#{$icon}"]) {
+    --pfe-navigation__icon--Background: #{map-get($icons, #{$icon})} no-repeat;
+  }
+}
+
+.pfe-navigation {
+  &__wrapper {
+    position: relative;
+    z-index: 2;
+    width: 100%;
+    background-color: #{pfe-local(BackgroundColor)};
+    color: #{pfe-local(Color)};
+    width: 100%;
+  }
+  &__container {
+    margin: 0 auto;
+    width: #{pfe-local(Width)};
+    color: #{pfe-local(Color)};
+    z-index: 2;
+
+    // need to do something here for IE11 since we can't redefine the variable
+    @each $size in (md, lg, xl) {
+      @media screen and (min-width: pfe-breakpoint(#{$size})) {
+        width: calc(#{pfe-breakpoint(#{$size})} - #{map-get($pfe-vars, container-spacer) * 2});
+      }
+    }
+
+    display: flex;
+    flex-flow: row nowrap;
+    align-items: stretch;
+    justify-content: space-between;
+
+    padding: 0 calc(#{pfe-var(container-padding)} * 2);
```

Comment:
Mockups show 24px of spacing on edges of mobile menu:
![image](https://user-images.githubusercontent.com/456863/60461336-3ee53b00-9c14-11e9-822a-d0f90815ef08.png)

currently demo page shows 32px on edges:
![image](https://user-images.githubusercontent.com/456863/60461427-781dab00-9c14-11e9-997a-153d17783ca3.png)

Reply:
It does but we don't have a variable for 20px in the design system.  I opted for container-spacer * 2 because it was close.  It looks pretty close to me.  I also think you're looking at an outdated version because your navigation element shows a `slot="[object Object]"`.

Reply:
@castastrophe I still see `slot="[object Object]"` 
Can you send me instructions on what steps I might be missing? Here's what I'm doing:
```
git checkout pfe-navigation
git fetch origin && git pull origin && git status  
git show --oneline -s
4e9cd55b (HEAD -> pfe-navigation, origin/pfe-navigation) Replace Red Hat logo with generic placeholder
npm install
npm run start
```
(new terminal window)
```
npm run dev pfe-navigation
```

Reply:
This is resolved now